change/@ni-nimble-tokens-a6408b9a-6f2d-4d38-97e3-b6537160321c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@fredvisser can you update the guid on this line to trigger a full storybook rebuild?
https://github.com/ni/nimble/blob/main/packages/storybook/.storybook/preview.js#L108

Reply:
Could alternatively use this as a chance to play with the externals paths to get storybook to recognize the changes:

https://github.com/ni/nimble/blob/545ede6d162548226430e0bd85a6d40fa1282427/.github/workflows/main.yml#L94-L99

My guess is we need to make the path relative to either the storybookBuildDir or storybookBaseDir paths instead of relative to the repo root.

Reply:
I took the easy way out, and updated the GUID…